src/controllers/userController.js

```diff
@@ -11,6 +11,7 @@ config();
  */
 class Authentication {
   /**
+     * Eric MALABA & Placide IRANDORA
```

Comment:
what's this for?

Reply:
Oh! It should be @author Eric MALABA & Placide IRANDORA. I tried to indicate the developers who worked on that method/function. But, I'm gonna removed it if it is not necessary.